modules/47_regipol/regiCarbonPrice/postsolve.gms

```diff
@@ -455,9 +469,9 @@ loop((ttot,ext_regi,taxType,targetType,qttyTarget,qttyTargetGroup)$pm_implicitQt
 loop((ttot,ext_regi,taxType,targetType,qttyTarget,qttyTargetGroup)$pm_implicitQttyTarget(ttot,ext_regi,taxType,targetType,qttyTarget,qttyTargetGroup),
   if(sameas(taxType,"tax"),
     if(iteration.val lt 15,
-      p47_implicitQttyTargetTaxRescale(ttot,ext_regi,qttyTarget,qttyTargetGroup) = (1 + pm_implicitQttyTarget_dev(ttot,ext_regi,qttyTarget,qttyTargetGroup) ) ** 4;
```

Comment:
I did that because I got some `x**y` error when running it. And replacing `** 2` with `sqr()`  and `** 4` with sqr(sqr())` should avoid that.

Reply:
could you do a test to see if you avoid the issue by using instead `exp(log(a) * b) = a ** b`?
e.g. (non tested code:)
```
p47_implicitQttyTargetTaxRescale(ttot,ext_regi,qttyTarget,qttyTargetGroup) = exp(log(1 + pm_implicitQttyTarget_dev(ttot,ext_regi,qttyTarget,qttyTargetGroup)) * 4 );
```
According tests made by Michaja some time ago, this was a better approach for doing exponentials in GAMs with smaller values when a similar issue happened in the calibration:
https://github.com/remindmodel/remind/pull/970

And this formulation is more flexible for which exponent we choose to use in the future if we find the need to change the pace of the convergence update.

Reply:
The problem was that I got some a < 0 error. I think that was because I messed up the unit in the switch (Mt vs. Gt), and therefore the scripts thought it was very far off. So that would not be solved with `log(a)`.

Reply:
could you use instead of two `sqr`, instead the power function: `power(a,b)`
`p47_implicitQttyTargetTaxRescale(ttot,ext_regi,qttyTarget,qttyTargetGroup) = power(1 + pm_implicitQttyTarget_dev(ttot,ext_regi,qttyTarget,qttyTargetGroup),4);`

Reply:
done.